tests/atomistics/structure/test_high_index_surface.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from pyiron_atomistics.atomistics.structure.factory import StructureFactory
+
+
+class TestHighIndexSurface(unittest.TestCase):
+    def test_high_index_surface(self):
+        slab, h, s, k = StructureFactory().get_high_index_surface(element='Ni', crystal_structure='fcc',
```

Comment:
The structure `slab` is unused in the tests. You could check it's length/chemical formula for completeness.